.github/workflows/qmk_userspace_build.yml

```
@@ -73,7 +73,7 @@ jobs:
 
       - name: Build
         run: |
-          qmk userspace-compile -e DUMP_CI_METADATA=yes || touch .failed
+          qmk userspace-compile -e DUMP_CI_METADATA=yes -p || touch .failed
```

As the current use case is to use these steps unpinned, and the qmk_firmware fork could be "old", we probably need this to be backwards compatible.

Something like this should work:

```suggestion
          if qmk userspace-compile --help | grep -q "print-failures"; then
              EXTRA_ARGS="-p"
          fi
          qmk userspace-compile -e DUMP_CI_METADATA=yes $EXTRA_ARGS || touch .failed
```